src/applications/vaos/project-cheetah/redux/reducer.js

```diff
@@ -40,6 +45,13 @@ const initialState = {
     facilitiesStatus: FETCH_STATUS.notStarted,
     clinics: {},
     clinicsStatus: FETCH_STATUS.notStarted,
+    facilityId: null,
+    siteId: null,
```

Comment:
Are you using the facilityId and siteId anywhere? There's a facility id in the data object already, I think that's what should be used, generally.

Reply:
Nope, you're right